code/src/components/SuggestedResturant.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable linebreak-style */
+/*import React from 'react';
+
+export const SuggestedResturant = ({ suggestedResturant, setSuggestedResturant }) => {
+  const handleSuggestedResturantChange = (event) => {
+    setSuggestedResturant(event.target.value);
+  }
+
+  return (
+
+    <><h2>We therefor suggest you to book a table for two at {suggestedResturant}</h2><input
+      type="text"
+      onChange={handleSuggestedResturantChange}
+      value={suggestedResturant} />
+    </>
+  );
+}
```

Comment:
This one is the same as email, not imported to the app so I can't see it in the survey.